driver-core/src/main/com/mongodb/operation/TransactionOperation.java

```diff
@@ -66,31 +64,25 @@ public WriteConcern getWriteConcern() {
     @Override
     public Void execute(final WriteBinding binding) {
         isTrue("in transaction", binding.getSessionContext().hasActiveTransaction());
-        return withConnection(binding, new CallableWithConnection<Void>() {
-            @Override
-            public Void call(final Connection connection) {
-                executeWrappedCommandProtocol(binding, "admin", getCommand(), connection, writeConcernErrorTransformer());
-                return null;
-            }
-        });
+        return executeRetryableCommand(binding, "admin", null, new NoOpFieldNameValidator(),
```

Comment:
Re-use the retryable writes support in CommandOperationHelper

Reply:
Ack